package.json

```diff
@@ -8,7 +8,7 @@
       "path": "dist/rudder-analytics.min.js",
       "import": "{ createComponent }",
       "gzip": true,
-      "limit": "38 kB"
+      "limit": "38.5 kB"
```

Comment:
Please check the final size and update this value.

Reply:
Addressed